baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -131,6 +132,10 @@ def handle(self, message: confluent_kafka.Message) -> None:
 
                 blob: bytes = message.value()
 
+                if not blob:
+                    warnings.warn("empty message body", BytesWarning)
```

Comment:
`warnings.warn` will generate a warning which is separate from logging and only show up in the logs in debug mode and only the first time this ever happens, not subsequent ones. Is that intentional? Alternatively you could use `logger.warning()` which would generate a message at the WARNING level in the logs on every time this is hit.